docker-compose.yml

```diff
@@ -1,21 +1,20 @@
-version: '3.1'
-
 # AlphaDev casinomex database
 services:
-  casinomex_db:
-    image: mysql
+  db:
+    image: mariadb
     restart: unless-stopped
     environment:
-      MYSQL_ROOT_PASSWORD: timba2024
-      MYSQL_DATABASE: timba-data-base
```

Comment:
**🚨 suggestion (security):** Update database environment variables

Ensure that the new database credentials are properly secured and not exposed in version control. Consider using environment variables or secrets management for sensitive information.